content/raw/chat/online_users.md

```diff
@@ -61,10 +68,13 @@ preconditions:
 
 **Status Selector**
 
-- The user MUST be able to select his state as 'online' or 'offline'
-- If the user has 'online' as selected, then they MUST be shown online in the communities member list.
-- If the user has 'offline' as selected, then they MUST be shown offline for everyone within 5 minutes regardless of activity, even if the user sends new messages.
+- The user MUST be able to select his state as 'always online' or 'inactive' or 'set status automatically'
+- If the user has 'always online' as selected, then they MUST be shown online in the communities member list.
+- If the user has 'inactive' as selected, then they MUST be shown offline for everyone within 5 minutes regardless of activity, even if the user sends new messages.
+- If the user has selected `Set status automatically`, then they MUST be shown online in the communities member list.
```

Comment:
"- If the user has selected `Set status automatically`, then they MUST be shown online in the communities member list." is incorrect.

If a user has selected `Set status automatically` (which is the default setting), then when the user is active in Status the user should be indicated as being Online, and when the user stops being active in Status then the user should be indicated as being Inactive